client/my-sites/pages/virtual-page/style.scss

```diff
@@ -0,0 +1,55 @@
+.virtual-page {
```

Comment:
I wonder if it'd be better for `<VirtualPage>` to inherit CSS from `<Page>`. What I mean is that instead of `card virtual-page is-compact` we have something like this `card page is-compact is-virtual`. I understand if this might make things more complicated. 

The nice thing about this is that we don't need to duplicate some CSS rules, and can avoid small details like this:

Title hover color (virtual page):
![Screen Shot 2022-10-19 at 10 58 17 AM](https://user-images.githubusercontent.com/797888/196586871-0a5d5dc2-d7c0-452a-988e-bb231886b79c.png)

Title hover color ("real" page):
![Screen Shot 2022-10-19 at 10 58 45 AM](https://user-images.githubusercontent.com/797888/196586934-12285fb7-2c75-4a7c-8e61-b8b3efd66cfe.png)

Let me know what you think!

Reply:
It makes sense to me. I'll disable eslint and re-use the class name 👍